### DISCUSSION TITLE
Adding sales offer fields when linking sales order with offer

### DISCUSSION THREAD

**Discussion by hahyes** [YetiForceCompany/YetiForceCRM#14123] [How to]
Quick question,

I want to get fields from linked offer in sales order. Is there a hidden option? How to do that?

**Comment by hahyes** [comment]
Ok, I found a way to do this. You need to make mapping between offer and sales order, but it makes you need to use only Generate record option. If I would select offer in sales order creation, fields are not mapping. Is there a way to change it? 

If not, there is a perspective to add this in future as feature? It would make live much easier. If I'm choosing offer to link with order, data should be mapped and updated in form.

**Reply by vovpff** [reply]
@hahyes check this file app_data/moduleHierarchy.php.
